modules/assim.sequential/inst/anchor/IC_prep_anchorSites.Rmd

```diff
@@ -63,10 +84,10 @@ for (i in seq_along(LAI_df$LAI_Output$site_id)) {
 
 ```{r}
 #prepare AGB
-AGB_IC <- PEcAn.data.remote::Prep_AGB_IC_from_2010_global(site.info = site_info, 
-                                                          paths.list = list(mean = "/projectnb/dietzelab/dongchen/anchorSites/IC/aboveground_biomass_carbon_2010.tif",
+AGB_IC <- PEcAn.data.remote::Prep_AGB_IC_from_geotiff(site_info = site_info, 
```

Comment:
this seems to revert to the old function name

Reply:
Fixed.